packages/block-library/src/cover/transforms.js

```diff
@@ -17,6 +17,7 @@ const transforms = {
 				createBlock(
 					'core/cover',
 					{
+						backgroundType: IMAGE_BACKGROUND_TYPE,
```

Comment:
Oh, so this is needed for the logic [here](https://github.com/WordPress/gutenberg/pull/36406/files#diff-c9b6422f332979151b7228eb2af812e84859338e6b4d5ef38e5341ef6e8f5a39R397) to take place, right? I wonder why it was missing? Maybe just an oversight?

Reply:
> Oh, so this is needed for the logic here to take place, right?

Yep!

> I wonder why it was missing? Maybe just an oversight?

I'd say so, yeah 🤷

Reply:
> I wonder why it was missing? Maybe just an oversight? 

Because the `image` type was the default it didn't need to be set previously